src/main/java/com/google/javascript/gents/TypeScriptGenerator.java

```diff
@@ -225,6 +225,19 @@ public CodeGenerator getCodeGenerator(Format outputFormat, CodeConsumer cc) {
                 .setOutputTypes(true)
                 .build();
 
+        // For whatever reason closure always prefixes the emit with an empty new line.
+        // Technically, this will not respect if the original .js intentionally had a new line
```

Comment:
Comments are out-of-date. We now respect the intentional newlines in the original .js files.

Reply:
duh, fixed.